### PR TITLE
fix bug with NPE when dev.client.properties is not found

### PR DESCRIPTION
**Pull request by KutuzovGregory**

### CODE REVIEW

**Comment by rmakulov** [comment]
Yes, it will be useful, thank you!

**Comment by oik77** [comment]
ok

**Comment by unrealwork** [comment]
:+1:

File: src/test/java/com/axibase/tsd/api/Config.java [hunk 1]
```diff
@@ -50,19 +51,25 @@ public static Logger getLogger() {
...
+            if (tryInitConfig(DEFAULT_CONFIG_FILE)) return instance;
+            throw new FileNotFoundException("*.client.properties not found");
         }
         return instance;
     }
 
+    private static boolean tryInitConfig(String config) {
+        URL configUrl = Config.class.getClassLoader().getResource(config);
+        if (configUrl != null) {
+            logger.debug("Using {} for config", config);
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/Config.java]: Maybe write "trying to use" instead?

**Pull request merged**